lib/pbench/test/unit/server/test_unpack_tarballs.py

```diff
@@ -18,10 +19,39 @@ def make_logger(server_config):
     return get_pbench_logger("TEST", server_config)
 
 
+class MockConfig:
+    ARCHIVE = Path("/mock/ARCHIVE")
+    INCOMING = Path("/mock/INCOMING")
+    RESULTS = Path("/mock/RESULTS")
+    TS = "my_timestamp"
+
+    def get(section: str, option: str, fallback=None) -> str:
+        """returns value of the given `section` and `option`"""
+
+        _sections = {
+            "pbench-server": {
+                "unpacked-states": "TO-INDEX",
+            },
+        }
+
+        if section not in _sections:
+            raise NoSectionError(section)
+        if option not in _sections[section]:
+            raise NoOptionError(option, section)
+        value = _sections[section][option]
+        if not value:
+            if not fallback:
+                raise ValueError(option, section)
+            else:
+                return fallback
+        return _sections[section][option]
```

Comment:
This is somewhat more general than what I had in mind, but it should be good.

Although, I would offer one tweak:  at line 41 you captured the value that you want to return at line 47.  (Also, I would omit the `else` at line 45.)
```
        ...
        value = _sections[section][option]
        if not value:
            if not fallback:
                raise ValueError(option, section)
            value = fallback  # Or, just return fallback here -- it's a style thing.
        return value
```